frontend_lib/src/component/Timeline/AddFileToUploadButton.jsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types'
 import { translate } from 'react-i18next'
 import PopupUploadFile from '../../container/PopupUploadFile.jsx'
 import IconButton from '../Button/IconButton.jsx'
+import { t } from 'i18next/dist/commonjs'
```

Comment:
```suggestion
```

Your IDE probably added this line automatically when your were typing

Reply:
yes it did, i'll remove it